testgrid/cluster/BUILD.bazel

```diff
@@ -21,7 +21,6 @@ k8s_objects(
     objects = [
         ":testgrid-config",
```

Comment:
Why not delete all of testgrid/cluster here? k8s doesn't run configurator as a pod; only as a postsubmit prow job.

Reply:
I think that's fair, but just in case I'd like to delete configurator-related stuff in a follow-up PR.